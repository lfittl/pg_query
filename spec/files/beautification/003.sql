SELECT a,b FROM c join d using (x) left outer join e on c.y = e.y, t WHERE c.h AND e.k = 123;
