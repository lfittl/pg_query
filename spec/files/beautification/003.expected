SELECT
  "a",
  "b"
FROM
  "c"
  JOIN "d" USING ("x")
  LEFT JOIN "e" ON ("c"."y" = "e"."y"),
  "t"
WHERE
  "c"."h" AND
  "e"."k" = 123
